docs/docs/customization.md

```diff
@@ -72,12 +72,12 @@ take place when:
 
 ### Swapping out components
 
-The collector, scanner, and eraser components can all be swapped out. This
+The collector, scanner, and remover components can all be swapped out. This
```

Comment:
i think we need to change these in version docs?

Reply:
Good catch

Reply:
should I change in both docs/ and version-v1.0.x/ or just version-v1.0.x/?

Reply:
oh my mistake, `docs` becomes the vNext docs